### ISSUE TITLE
Program Areas: Update alt text for .page-card-image to adhere to WCAG

### ISSUE THREAD

**Issue by MattPereira** [hackforla/website#3875]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.

### Overview
As a developer, we need to provide clear and descriptive alt text on the `Program Areas` page so that we adhere to the Web Content Accessibility Guidelines (WCAG) and we can achieve our mission of inclusivity.

### Action Items
- [x] Change the image alt property value within `pages/program-areas.html`:

**From:**
```
<img class="page-card-image" src="{{ program_areas[1].image }}" alt="{{ program_areas[1].image_alt }}" />
```

**To:**
```
<img class="page-card-image" src="{{ program_areas[1].image }}" alt="" />
```

- [x] Ensure that the corresponding `Program Areas` page stays the same after the change
- [x] Using developer tools to inspect the image, ensure that the new alt text gets incorporated after the change.
  - **Note:** `alt=""` in code while show up as `alt` when using developer tools to inspect the image's alt text property.

### Resources/Instructions
- https://www.hackforla.org/program-areas

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: `pages/program-areas.html`
- Click on the filename below to see the source file code:
https://github.com/hackforla/website/blob/075ce5f7d67c7732e258148b0f4a1a9342095e26/pages/program-areas.html#L24

#### Background information about the Jekyll framework (optional reading)
- [Jekyll Data Files](https://jekyllrb.com/docs/datafiles/)
- [Jekyll Front Matter](https://jekyllrb.com/docs/front-matter/)

#### Information about WCAG and alt text (optional reading)
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

**Comment by mademarc** [comment]
i. Availability: feb 10
ii. ETA: 5:30pm

**Comment by mademarc** [comment]
Provide Update
1. Progress:  working on pushing the code & update the repo
2. Blockers: No blockers
3. Availability: Will get inti this matter in 3-5 business days
4. ETA: Should be done by 03/09/2023
